allauth/mfa/views.py

```diff
@@ -236,6 +242,7 @@ def form_valid(self, form):
         signals.authenticator_reset.send(
             sender=Authenticator, user=self.request.user, authenticator=rc_auth.instance
         )
+        adapter.send_notification_mail("mfa/email/totp_reset", self.request.user, {})
```

Comment:
It's not the TOTP authenticator that is reset. The recovery codes are regenerated. I think we need to switch to:

`mfa/email/recovery_codes_generated`

Reply:
replaced and renamed